packages/angular/src/wallet/wallet.store.ts

```diff
@@ -0,0 +1,295 @@
+import { Inject, Injectable, Optional } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { SendTransactionOptions, WalletNotConnectedError, WalletNotReadyError } from '@solana/wallet-adapter-base';
+import { WalletName } from '@solana/wallet-adapter-wallets';
+import { Connection, Transaction } from '@solana/web3.js';
+import { asyncScheduler, combineLatest, defer, from, Observable, of, throwError } from 'rxjs';
+import { catchError, concatMap, filter, first, map, observeOn, switchMap, tap, withLatestFrom } from 'rxjs/operators';
+
+import { fromAdapterEvent, isNotNull } from '../operators';
+import { WalletNotSelectedError } from './wallet.errors';
+import { WALLET_CONFIG } from './wallet.tokens';
+import { WalletConfig, WalletState } from './wallet.types';
+
+export const WALLET_DEFAULT_CONFIG: WalletConfig = {
+    wallets: [],
+    autoConnect: false,
+    localStorageKey: 'walletName',
+    onError: (error: unknown) => console.error(error),
+};
+
+@Injectable()
+export class WalletStore extends ComponentStore<WalletState> {
+    private readonly _autoConnect = this._config?.autoConnect || false;
+    private readonly _localStorageKey = this._config?.localStorageKey || 'walletName';
+    private logError = this._config?.onError || ((error: unknown) => console.error(error));
+    readonly wallets$ = this.select((state) => state.wallets);
+    readonly selectedWallet$ = this.select((state) => state.selectedWallet);
+    readonly connected$ = this.select((state) => state.connected);
+    readonly connecting$ = this.select((state) => state.connecting);
+    readonly disconnecting$ = this.select((state) => state.disconnecting);
+    readonly wallet$ = this.select((state) => state.wallet);
```

Comment:
Can we change `selectedWallet` to `name` or `walletName`? `selectedWallet` implies that it's the same type as `wallet`, and keeping some parity with the names in the react package will probably help us maintain this easier when either changes.

Reply:
You're definitely right. I think `name` is the way to go so it matches the react package